e2e/app/component/tiered-menu.ts

```diff
@@ -1,22 +1,24 @@
 import Container from 'app/container';
 import {ElementHandle, Page} from 'puppeteer';
+import {waitWhileLoading} from 'utils/test-utils';
 
-const defaultXpath = '//*[contains(concat(" ", normalize-space(@class)), " p-menu-overlay-visible")]';
+const defaultXpath = '//*[contains(concat(" ", normalize-space(@class), " "), " p-menu-overlay-visible ")]';
 
+// Cascading menus
 export default class TieredMenu extends Container {
 
   constructor(page: Page, xpath: string = defaultXpath) {
     super(page, xpath);
   }
 
   /**
-   * Select menu item(s).
-   * @param {string[]} menuItems
+   * Select options in menu.
+   * @param {string[]} options
    */
-  async clickMenuItem(menuItems: string[]): Promise<void> {
+  async select(options: string[]): Promise<void> {
```

Comment:
IMO, `select` is a better name for describing action of selecting option in menu.